### DISCUSSION TITLE
建议 一定时间没有播放音乐取消网易云状态

### DISCUSSION THREAD

**Discussion by chenchen7427** [aliencaocao/netease_cloudmusic_discord_rpc#34] [Ideas]
一个小建议 就是不知道能不能实现比如半个小时没有播放音乐 就取消discord上网易云的状态

**Comment by aliencaocao** [comment]
可以实现，已经加到计划功能清单https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/issues/35

**Reply by aliencaocao** [reply]
功能已经实现 https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/releases/tag/0.3.5
